sdk/nodejs/ltm/monitor.ts

```diff
@@ -16,13 +16,12 @@ import * as utilities from "../utilities";
  * import * as pulumi from "@pulumi/pulumi";
  * import * as f5bigip from "@pulumi/f5bigip";
  * 
- * const bigip_ltm_monitor_monitor = new f5bigip.ltm.Monitor("monitor", {
+ * const monitor = new f5bigip.ltm.Monitor("monitor", {
  *     destination: "1.2.3.4:1234",
- *     interval: Number.parseFloat("999"),
- *     name: "/Common/terraform_monitor",
```

Comment:
FYI @pgavlin I do think these are cases where the `name` really must be set to a name of this specific format (and hence where auto-naming will not actually be appropriate for the `name` property.

Reply:
> FYI @pgavlin I do think these are cases where the `name` really must be set to a name of this specific format (and hence where auto-naming will not actually be appropriate for the `name` property.

This is why I was reluctant to add the "elide names" feature without a schema change. As it stands, the schema that `tf2pulumi` receives does not indicate whether or not a particular property is auto-named. We could do a much better job of things--including detecting this sort of case--if this information was present.

That said, no resources in this provider are auto-named. In the short term, we could update its `Makefile` to add `--autoname=""` to its `tfgen` invocation, which will disable `tf2pulumi`'s name elision.

Linking the original issue for reference: https://github.com/pulumi/tf2pulumi/issues/43

Reply:
> In the short term, we could update its `Makefile` to add `--autoname=""` to its `tfgen` invocation, which will disable `tf2pulumi`'s name elision.

That seems like a good solution for providers (like this one) which don't do global `name` autonaming.